ansible/requirements.yml

```diff
@@ -123,6 +123,10 @@
 - src: ome.versioncontrol_utils
   version: 1.1.0
 
+# this should be updated after merging the PRs and publich the role
+- name: ome.omero_ms_image_region
+  src: https://github.com/khaledk2/ansible-role-omero-ms-image-region/
+  version: new_ms_release
```

Comment:
For the `prod122` update we can certainly use the in-progress version but for future deployments this role will need to be released in order to merge this step